ui/components/multichain/import-tokens-modal/import-tokens-modal.js

```diff
@@ -137,7 +137,7 @@ export const ImportTokensModal = ({ onClose }) => {
   const accounts = useSelector(getInternalAccounts);
   const tokens = useSelector((state) => state.metamask.tokens);
   const rpcPrefs = useSelector(getRpcPrefsForCurrentProvider);
-  const contractExchangeRates = useSelector(contractExchangeRateSelector);
+  const contractExchangeRates = useSelector(getTokenExchangeRates);
```

Comment:
I don't think the previous selector made sense.  It only returned the exchange rate of the current token being transacted.  Whereas the variable is accessed like an object containing rates for multiple tokens.